### ISSUE TITLE
Subsumption Checking

### ISSUE THREAD

**Issue by Demirrr** [dice-group/Ontolearn#264]
We weren't quite sure about the following subsumtion checking

`Tbox={D\subseteq C, C\subseteq D}` and `Abox=\emptyset and K=(Tbox, Abox)`

In the closed word assumption, is the following axiom `K \entails D \equiv C`?

1. `get_equivalent_classes(input=D)={C}`
2. 1. `get_equivalent_classes(input=D)=\emptyset`

**Comment by Demirrr** [comment]
@ngonga stated that `K \entails D \equiv C` holds.

**Comment by Demirrr** [comment]
@alkidbaci can you confirm whether this issue has been solved?

**Comment by alkidbaci** [comment]
Yes I'll try to merge everything today, also issue #263 . I will close these issues as soon as I do that.
